index.md

```diff
@@ -1,4 +1,127 @@
+          <a href="https://github.com/aspnet/Blazor/wiki/FAQ" class="btn btn-primary">What is Blazor?</a>
+        </section>
+      </div>
+    </div>
+  </div>
+</div>
+<div class="counter-key-section">
+  <div class="container">
+    <div class="row">
+      <div class="col-md-8 col-md-offset-2">
+        <section>
+          <h2>Full-Stack .NET</h2>
+          <p class="lead">Do full-stack .NET development using stable and consistent tools, languages and APIs both in the browser and on the server.</p>
+          <a href="https://www.microsoft.com/net" class="btn btn-primary">Learn more about the .NET platform</a>
+        </section>
+        <i class="glyphicon glyphicon-tasks"></i>
+      </div>
+    </div>
+  </div>
+</div>
+<div class="key-section">
+  <div class="container content">
+    <div class="row">
+      <div class="col-md-8 col-md-offset-2">
+        <i class="glyphicon glyphicon-globe"></i>
+        <section>
+          <h2>Runs in all browsers and implements .NET Standard</h2>
+          <p class="lead">Blazor runs in all browsers on the real .NET runtime with full support for [.NET Standard](https://docs.microsoft.com/dotnet/standard/net-standard). Blazor requires no plugins and no code transpilation, only open web standards.</p>
+          <a href="/docs/introduction/index.html" class="btn btn-primary">How Blazor works</a>
+        </section>
+      </div>
+    </div>
+</div>
+<div class="counter-key-section">
+  <div class="container">
+    <div class="row">
+      <div class="col-md-8 col-md-offset-2">
+        <section>
+          <h2>Native performance with WebAssembly</h2>
+          <p class="lead">Runs on WebAssembly, giving you native performance in the browser and a trusted security sandbox.</p>
+        </section>
+        <i class="glyphicon glyphicon-fire"></i>
+      </div>
+    </div>
+  </div>
+</div>
+<div class="key-section">
+  <div class="container content">
+    <div class="row">
+      <div class="col-md-8 col-md-offset-2">
+        <i class="glyphicon glyphicon-transfer"></i>
+        <section>
+          <h2>Native browser apps</h2>
+          <p class="lead">Easily interact with your existing JavaScript code, your favorite libraries, and any browser API through JavaScript interop.</p>
+        </section>
+      </div>
+    </div>
+</div>
+<div class="counter-key-section">
+  <div class="container">
+    <div class="row">
+      <div class="col-md-8 col-md-offset-2">
+        <section>
+          <h2>Simple and productive</h2>
+          <p class="lead">Get started fast and remain productive with project templates, great tooling, and reusable components.</p>
+          <a href="https://marketplace.visualstudio.com/items?itemName=aspnet.blazor" class="btn btn-primary">Tools for Blazor</a>
+        </section>
+        <i class="glyphicon glyphicon-console"></i>
+      </div>
+    </div>
+  </div>
+</div>
+<div class="key-section">
```

Comment:
I think we can cut this section. It doesn't really add much.

Reply:
Not clear from the GitHub ui which specific section you want to cut. Can you give the headline for the section? "Get Involved", "Simple and productive", or other?

Reply:
Cut "Get Involved"